diff --git a/node_modules/@nestjsx/crud-typeorm/lib/typeorm-crud.service.d.ts b/node_modules/@nestjsx/crud-typeorm/lib/typeorm-crud.service.d.ts
index a8e9028..a045455 100644
--- a/node_modules/@nestjsx/crud-typeorm/lib/typeorm-crud.service.d.ts
+++ b/node_modules/@nestjsx/crud-typeorm/lib/typeorm-crud.service.d.ts
@@ -6,13 +6,13 @@ export declare class TypeOrmCrudService<T> extends CrudService<T> {
     private entityColumns;
     private entityPrimaryColumns;
     private entityColumnsHash;
-    private entityRelationsHash;
+    protected entityRelationsHash;
     constructor(repo: Repository<T>);
     readonly findOne: Repository<T>['findOne'];
     readonly find: Repository<T>['find'];
     readonly count: Repository<T>['count'];
     private readonly entityType;
-    private readonly alias;
+    protected readonly alias;
     getMany(req: CrudRequest): Promise<GetManyDefaultResponse<T> | T[]>;
     getOne(req: CrudRequest): Promise<T>;
     createOne(req: CrudRequest, dto: DeepPartial<T>): Promise<T>;
@@ -30,9 +30,9 @@ export declare class TypeOrmCrudService<T> extends CrudService<T> {
     private getOneOrFail;
     private getOneShallowOrFail;
     private prepareEntityBeforeSave;
-    private getAllowedColumns;
-    private getRelationMetadata;
-    private setJoin;
+    protected getAllowedColumns;
+    protected getRelationMetadata;
+    protected setJoin;
     private setAndWhere;
     private setOrWhere;
     private setSearchCondition;
@@ -45,5 +45,5 @@ export declare class TypeOrmCrudService<T> extends CrudService<T> {
     private getSort;
     private getFieldWithAlias;
     private mapSort;
-    private mapOperatorsToQuery;
+    protected mapOperatorsToQuery;
 }
